### ISSUE TITLE
Dev server fails to rebuild dependencies

### ISSUE THREAD

**Issue by ryan-edge-ck** [ionic-team/stencil#102]
## Description of Problem
When using 3rd party dependencies in components, the dev server occasionally cannot build 3rd party dependencies

## Expected Behavior
Rebuild is successful

## Actual Result
On every other save, the rebuild fails, resulting in a "Cannot find module '<npm module>' error."

![image](https://user-images.githubusercontent.com/28316004/29675584-94874924-88c3-11e7-96ba-3fd8d98718cf.png)

Refer to [example](https://github.com/chimon2000/stencil-issues/tree/issue-102) to reproduce issue

**Comment by zwacky** [comment]
I ran into the same issue (similar to #98). Then I revisited their configuration:

> Since everything in Stencil is async and lazy loaded by default, it is important to NOT have any hard import statements linking components together. 

https://stenciljs.com/docs/stencil-config

So what I understand for now is to include a `<script src="build/vendor.js"></script>` alongside with all needed 3rd party libraries.

I was briefly thinking of a utility item in the *stencil.config.js* like `vendor: true`, that will automatically strip out all imports coming from node_modules (except @stencil/**), tree-shake them and pack it into a vendor.js.
I'm too fresh into stencil to know if that's the way stencil should go in the future.
